VimeoNetworking/Sources/Request+PasswordProtected.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Request+PasswordProtected.swift
+//  VimeoNetworking
+//
+
+extension Request {
+    
+    /// Returns a new request for a password-protected item, such as a video or an album.
+    ///
+    /// - Parameters:
+    ///   - uri: The URI for the item.
+    ///   - password: The password for the item that will be sent to the server.
+    /// - Returns: Returns a `Request` for a password-protected item.
+    public static func passwordProtectedRequest(for uri: String, password: String) -> Request {
+        let parameters = ["password": password]
```

Comment:
nit since these are both extensions on request anyway we can share "password" as a static constant

Reply:
I actually made a mistake putting the Constants struct in `Request+Album`. I moved it into here, which should make more sense. I can't add constants to an extension for a generic type so the best I think I can do is to make the `Constants` struct `fileprivate` and include it outside of the extension.